clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -202,7 +203,7 @@ public AbstractCoordinator(LogContext logContext,
      */
     protected abstract Map<String, ByteBuffer> performAssignment(String leaderId,
                                                                  String protocol,
-                                                                 Map<String, ByteBuffer> allMemberMetadata);
+                                                                 List<JoinGroupResponseData.JoinGroupResponseMember> allMemberMetadata);
```

Comment:
Should this be a `JoinGroupResponseDataSet`?

Reply:
I don't see this struct anywhere, but I think a list should be fine here?

Reply:
Sorry, I meant `JoinGroupResponseMemberSet`.  Anyway, a list is fine too-- I don't feel that strongly about it.